### PR TITLE
Improving Matrix3d.ts: Part 5 (test - line 1 to 500)

### PR DESCRIPTION
**Pull request by saeeedtorabi**
- improved tests formatting and readability

### CODE REVIEW

File: core/geometry/src/test/geometry3d/FrameBuilder.test.ts [hunk 1]
```diff
@@ -218,13 +219,15 @@ describe("FrameBuilder", () => {
...
+          MatrixTests.checkProperties(
+            ck,
             localToWorld.matrix,
             select === AxisScaleSelect.Unit,  // unit axes in range are identity
             select === AxisScaleSelect.Unit,  // and of course unitPerpendicular
             select === AxisScaleSelect.Unit,  // and of course rigid.
             true, // always invertible
-            true);  // always diagonal
+            true
+          );  // always diagonal
```
Comment (dassaf4) [inline core/geometry/src/test/geometry3d/FrameBuilder.test.ts:230]: This comment ("always diagonal") belongs to the last arg, so I'd move it up a line (to 229).

File: core/geometry/src/test/geometry3d/Matrix3d.test.ts [hunk 2]
```diff
@@ -18,160 +18,172 @@ import * as bsiChecker from "../Checker";
...
+      MatrixTests.checkProperties(ck, frame2, false, true, false, true, false);
+
+      let vector;
+      const e = 1.0 / 64.0;
+      for (vector of [
+        Vector3d.create(1, 2, 3),
+        Vector3d.create(1, 0, 0),
+        Vector3d.create(0, 1, 0),
+        Vector3d.create(0, 0, 1),
+        Vector3d.create(e, e, 3.0), // triggers near-z logic
```
Comment (dassaf4) [inline core/geometry/src/test/geometry3d/Matrix3d.test.ts:113]: Looks like `Vector3d.create(e, e, 0.5)` test was lost.

**Review by dassaf4 — APPROVED** [review]

File: core/geometry/src/test/geometry3d/FrameBuilder.test.ts (hunk 1 above)
Comment (saeeedtorabi) [reply core/geometry/src/test/geometry3d/FrameBuilder.test.ts:230]: will fix

File: core/geometry/src/test/geometry3d/Matrix3d.test.ts (hunk 2 above)
Comment (saeeedtorabi) [reply core/geometry/src/test/geometry3d/Matrix3d.test.ts:113]: I intentionally removed it as it does serve the same purpose as (e, e, 3.0)

**Pull request merged**